modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -142,6 +147,10 @@ def claimant_type_other_text
       data_attributes['claimantTypeOtherValue']&.strip
     end
 
+    def potential_pact_act
+      data_attributes&.dig('potentialPactAct') ? true : false
+    end
```

Comment:
@caseywilliams This doesn't look like it was supposed to make it in.

Reply:
Thanks, I was focused on the merge conflicts and totally missed this